presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +22,363 @@ connection properties as appropriate for your setup:
+      - ``CEILING`` - Rounding mode to round towards
+        positive infinity.
+      - ``FLOOR`` - Rounding mode to round towards negative
+        infinity.
+      - ``HALF_DOWN`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding down is used.
+      - ``HALF_EVEN`` - Rounding mode to round towards the
+        ``nearest neighbor`` unless both neighbors are
+        in which case rounding towards the nearest neighbor is
+        performed.
+      - ``HALF_UP`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding up is used
+      - ``UP`` - Rounding mode to round towards zero.
+      - ``DOWN`` - Rounding mode to round towards zero.
+
+    - ``UNNECESSARY``
+
+Oracle to Presto type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presto supports selecting Oracle database types. This table shows the Oracle to
+Presto data type mapping:
+
+
+.. list-table:: Oracle to Presto type mapping
+  :widths: 20, 20, 60
+  :header-rows: 1
+
+  * - Oracle database type
+    - Presto type
+    - Notes
+  * - ``NUMBER(p, s)``
+    - ``DECIMAL(p, s)``
+    -  See :ref:`number mapping`
+  * - ``NUMBER(p)``
+    - ``DECIMAL(p, 0)``
+    - See :ref:`number mapping`
+  * - ``FLOAT[(p)]``
+    - ``DOUBLE``
+    -
+  * - ``BINARY_FLOAT``
+    - ``REAL``
+    -
+  * - ``BINARY_DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``VARCHAR2(n CHAR)``
+    - ``VARCHAR(n)``
+    -
+  * - ``VARCHAR2(n BYTE)``
+    - ``VARCHAR(n)``
+    -
+  * - ``NVARCHAR2(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``NCHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``CLOB``
+    - ``VARCHAR``
+    -
+  * - ``NCLOB``
+    - ``VARCHAR``
+    -
+  * - ``RAW(n)``
+    - ``VARBINARY``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``TIMESTAMP``
+    - See :ref:`datetime mapping`
+  * - ``TIMESTAMP(p)``
+    - ``TIMESTAMP``
+    - See :ref:`datetime mapping`
+  * - ``TIMESTAMP(p) WITH TIME ZONE``
+    - ``TIMESTAMP WITH TIME ZONE``
+    - See :ref:`datetime mapping`
+
+If an Oracle table contains a type that's not listed in the above table, an
+error shows:
+
+ - An ``unsupported-type.handling-strategy`` equals ``FAIL``
+   (default configuration), any queries on the table fail.
+ - An ``unsupported-type.handling-strategy`` equals ``IGNORE``
+   (default configuration), Presto no longer accesses the column.
+ - An ``unsupported-type.handling-strategy`` equals ``CONVERT_TO_VARCHAR``
+   (default configuration), a column value converts to an unbounded ``VARCHAR``.
+
+Presto to Oracle type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presto supports creating tables with the following types in an Oracle database.
+The table shows the mappings from Presto to Oracle data types:
+
+.. note::
+   Oracle connectors can't write Presto types that aren't listed in the table.
```

Comment:
I think it should say that for types not listed below Presto cannot perform `CREATE TABLE ... AS SELECT ..` operation. Inserting into alerady existing table uses `Oracle to Presto` type mapping.

cc: @findepi are there any quirks here?

Reply:
I edited based on your note, lemme know what you think of the new version.